test/micro/org/openjdk/bench/java/util/ArraysMismatchPartialInlining.java

```diff
@@ -38,6 +39,9 @@
 @BenchmarkMode(Mode.Throughput)
 @OutputTimeUnit(TimeUnit.MILLISECONDS)
 @State(Scope.Thread)
+@Warmup(iterations = 4, time = 2)
+@Measurement(iterations = 4, time = 2)
+@Fork(value = 3)
 public class ArraysMismatchPartialInlining {
 
     @Param({"3", "4", "5", "6", "7", "15", "31", "63", "95", "800"})
```

Comment:
Can we prune a few of these? At least 4, 5, 6 seem like they'll not add much signal.